lib/Service/ExternalFolderService.php

```diff
@@ -525,7 +525,7 @@ public function updateMetadataFile(
 				throw new MetadataException('metadata file not found');
 			}
 
-			$metadataFile = $folder->newFile(MetadataService::METADATA_FILE, '');
+			$metadataFile = $folder->newFile(MetadataService::METADATA_FILE, json_encode($point, JSON_PRETTY_PRINT));
```

Comment:
can you also returns `$metadataFile` just after its creation ?

Reply:
Done.